airbyte-db/db-lib/src/main/java/io/airbyte/db/instance/configs/migrations/V0_40_11_002__AddSchemaChangeColumnsToConnections.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.db.instance.configs.migrations;
+
+import java.util.Arrays;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.jooq.DSLContext;
+import org.jooq.EnumType;
+import org.jooq.impl.DSL;
+import org.jooq.impl.SQLDataType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class V0_40_11_002__AddSchemaChangeColumnsToConnections extends BaseJavaMigration {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(V0_40_11_002__AddSchemaChangeColumnsToConnections.class);
+
+  @Override
+  public void migrate(final Context context) throws Exception {
+    LOGGER.info("Running migration: {}", this.getClass().getSimpleName());
+
+    final DSLContext ctx = DSL.using(context.getConnection());
+
+    addNonBreakingChangePreferenceEnumTypes(ctx);
+
+    addNotifySchemaChanges(ctx);
+    addNonBreakingChangePreference(ctx);
+    addBreakingChange(ctx);
+  }
+
+  private static void addNonBreakingChangePreferenceEnumTypes(final DSLContext ctx) {
+    ctx.createType(NonBreakingChangePreferenceType.NAME)
+        .asEnum(Arrays.stream(NonBreakingChangePreferenceType.values()).map(NonBreakingChangePreferenceType::getLiteral).toList())
+        .execute();
+  }
+
+  private static void addNotifySchemaChanges(final DSLContext ctx) {
+    ctx.alterTable("connection")
+        .addColumnIfNotExists(DSL.field(
+            "notify_schema_changes",
+            SQLDataType.BOOLEAN.nullable(false).defaultValue(true)))
+        .execute();
+  }
+
+  private static void addNonBreakingChangePreference(final DSLContext ctx) {
+    ctx.alterTable("connection")
+        .addColumnIfNotExists(DSL.field(
+            "non_breaking_change_preference",
+            SQLDataType.VARCHAR.asEnumDataType(NonBreakingChangePreferenceType.class).nullable(false)
+                .defaultValue(NonBreakingChangePreferenceType.IGNORE)))
+        .execute();
+
+  }
+
+  private static void addBreakingChange(final DSLContext ctx) {
+    ctx.alterTable("connection")
+        .addColumnIfNotExists(DSL.field(
+            "breaking_change",
```

Comment:
I think this is about notifications, can we be more explicit and add it to the name?

Reply:
this is not about notifications - this is a boolean about whether this connection has breaking changes